pontoon/base/views.py

```diff
@@ -158,9 +159,51 @@ def projects(request, template='projects.html'):
     return render(request, template, data)
 
 
-def translate(request, locale, slug, part=None, template='translate.html'):
+def locale_project(request, locale, slug):
+    """Locale-project overview."""
+
+    # Redirect to translate view
+    if request.GET.get('q'):
+        return translate(request, locale, slug, '')
+
+    try:
+        l = Locale.objects.get(code__iexact=locale)
```

Comment:
Check out [one of my favorite shortcut functions](https://docs.djangoproject.com/en/1.8/topics/http/shortcuts/#get-object-or-404).